docs/src/icons/SvgDiscord.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
+import RootSvg, { RootSvgProps } from 'docs/src/icons/RootSvg';
```

Comment:
I think it should use SvgIcon here.

Reply:
I'm having issues using `SvgIcon`. I don't think it's properly set up in `/layout` folder path because every time I import SvgIcon, I get a weird error for the `fontSize` prop. I even tried overriding the default prop but still got an error. I also so don't see any use-cases of SvgIcon in the `/layout` folder, which makes me believe that the person who implemented the StackOverflow icon on the footer had the same issue as me. Im using the same convention as `SvgStackOverflow` in `docs/src/icons/SvgStackOverflow`.

The proper way this should be handled is by adding these icons in our official [Material UI Icons](https://mui.com/material-ui/material-icons/), but saw that it is currently not in our scope:

https://github.com/mui/material-ui/issues/35100
https://github.com/mui/material-ui/issues/19371
https://github.com/mui/material-ui/issues/35218

The Discord blog and Tweet will be going out this week and would like the footer to be ready by then. I can address the Material UI Icon and/or the SvgIcon issue after if that's okay.

Reply:
I pushed a change with what I had in mind. It's more using Material UI "by the book".